src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
+ *
+ */
+public class Recommendation {
+	private Preference correlatedPreference;
+	private UserProfile user;
```

Comment:
Do we really need to store the user? I understand that there is an implicit linkage, but is getUser() ever called by something that wouldn't already be concerned with only one user? I could see a Recommendation just being a Preference and a score at the heart of it. I can go either way, though.